yt/go/yt/interface.go

```diff
@@ -921,6 +927,35 @@ type AdminClient interface {
 		options *RemoveMemberOptions,
 	) (err error)
 
+	// http:verb:"set_user_password"
+	// http:params:"user","new_password_sha256","current_password_sha256"
```

Comment:
Instead of using AdminClient, it is better to create a new AuthClient for auth-related methods.

Reply:
done